go.mod

```diff
@@ -42,23 +42,23 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Jeffail/gabs v1.4.0
 	github.com/aws/aws-sdk-go v1.30.15
-	github.com/aws/aws-sdk-go-v2 v1.13.0 // indirect
-	github.com/aws/aws-sdk-go-v2/config v1.13.1 // indirect
-	github.com/aws/aws-sdk-go-v2/service/ec2 v1.29.0 // indirect
-	github.com/aws/smithy-go v1.10.0 // indirect
+	github.com/aws/aws-sdk-go-v2 v1.13.0
```

Comment:
Why is this getting changed? Didn't @sethAmazon already fix this by running `go mod tidy`? Can you please run that on your workspace too?

Reply:
Maybe I have not merged the changes from @sethAmazon yet. Moreover, I used `go mod tidy` to make this changes so properly does no different from @sethAmazon 's changes. However, making sure by pulling the latest changes.